iaso/api/org_unit_types/viewsets.py

```diff
@@ -3,20 +3,24 @@
 from rest_framework.response import Response
 
 from iaso.models import OrgUnitType
-from .serializers import OrgUnitTypeSerializer
+from .serializers import OrgUnitTypeSerializerV1, OrgUnitTypeSerializerV2
 from ..common import ModelViewSet
 
 
 class OrgUnitTypeViewSet(ModelViewSet):
-    """Org unit types API
+    """Org unit types API (deprecated)
+
+    This endpoint it deprecated, Use /v2/orgunittypes/ instead, this is kept only  for compatiblity with the mobile
+    application
 
+    Confusingly in this version  `sub_unit_types` map to allow_creating_sub_unit_types.
```

Comment:
Nit-picking, you didn't put the back quotes around "allow_creating_sub_unit_types"

```suggestion
    Confusingly in this version  `sub_unit_types` map to `allow_creating_sub_unit_types`.
```